source/MRViewer/MRViewer.cpp

```diff
@@ -365,9 +366,9 @@ int launchDefaultViewer( const Viewer::LaunchParams& params, const ViewerSetup&
         printCurrentTimerBranch();
         res = 1;
     }
-
-    return res;
 #endif
+    setup.unloadExtendedLibraries();
```

Comment:
better to call in `launchShut_` function

Reply:
Actually current position is even better, since it is after `launchShut`